modules/ballerina-core/src/test/java/org/ballerinalang/core/EnvironmentInitializer.java

```diff
@@ -38,6 +38,7 @@ public static BLangProgram setup(String sourcePath) {
         // Initialize server connectors before starting the test cases
         BallerinaConnectorManager.getInstance().initialize(new MessageProcessor());
         BallerinaConnectorManager.getInstance().registerServerConnectorErrorHandler(new TestErrorHandler());
+        BallerinaConnectorManager.getInstance().registerServerConnectorErrorHandler(new TestJmsErrorHandler());
```

Comment:
Since we have moved the error handler to ballerina level, it detects the default jms error handler. In our test case, we are checking the exceptions. But the default jms error handler does not throw any exceptions, but just logs the error. For testing purposes I have added this.